### PR TITLE
Feature/controllers

### PR DESCRIPTION
**Pull request by Krishnabot**
In this Pull Request 

- Made sure that the  app handles the URLs used in the given  wireframes:
blog_root_users
blog_single_post
blog_user_all_posts
blog_user_page

- For each URL

   - A route is created.
   - An action in the correct controller is created.
   - A view file is created.
   - View files include only placeholders written in plain HTML.

- Made sure that code is able to handle only the 4 required URLs.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,69 +0,0 @@
-class UsersController < ApplicationController
-  before_action :set_user, only: %i[show edit update destroy]
```
Comment (julie-ify) [inline app/controllers/users_controller.rb:2]: - [ ] Kindly restore this controller, as it is the correct controller you should have. Kindly keep only the `index` and `show` actions.

File: app/controllers/concerns/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show; end
+end
```
Comment (julie-ify) [inline app/controllers/concerns/users_controller.rb]: - [ ] Kindly delete this controller. It is not needed. The one mentioned above is the correct controller.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Krishnabot,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @Krishnabot    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**